wire-tests-swift/src/main/swift/Form.swift

```diff
@@ -180,13 +189,26 @@ public struct Form {
 
         public var unknownFields: Foundation.Data = .init()
 
-        public init() {
+        public init(configure: (inout Self) -> Void = { _ in }) {
```

Comment:
Yeah, we should remove the `configure` if there are no params.

Reply:
- [ ] Remove Configure when appropriate